pkg/controller/manager/clusternetwork/controller.go

```diff
@@ -137,6 +153,26 @@ func (h Handler) initializeClusterNetwork() error {
 	return nil
 }
 
+func (h Handler) initializeUntaggedNetwork() error {
+	untaggedNetworkConfig := fmt.Sprintf(`{"cniVersion":"0.3.1","name":"%s","type":"bridge","bridge":"%s-br","promiscMode":true,"ipam":{}}`,
```

Comment:
suggest to use a const to represent the cni version, and format it with that const